### ISSUE TITLE
Feature request: show application on lock screen

### ISSUE THREAD

**Issue by m0nkeyc0de** [thetwom/Tuner#95]
Hello, I really like to use your great application :)

Could you add an option to superpose the application on lock screen like map applications are doing during navigation ?

Use case: 
I'm using it during rehearsal sessions to check how much out-of-tune I am but I don't need it all the time. Therefore I turn the screen off and put my phone under my chair. When I need it again I take my phone back and I need to unlock it to continue to use the application, which takes some time. Of course I can't ask the musical director to wait until I unlock my phone. Having the application superposed to the lock screen would make it available almost instantly when turning the phone on again.

I guess this was added in Organics Maps in the following PR:
https://github.com/organicmaps/organicmaps/pull/1928

Regards

**Comment by thetwom** [comment]
Glad you like the app. Fully understand your use case and that it would be helpful here. At the moment I am not really sure, whether is what the average user would expect and if such a feature could also be annoying? Could be of course an option in the settings, but each extra setting makes the list of settings longer which is not necessarily good. 

I have to think about it a bit longer. Maybe I will give it a try how easy the Android system it makes to have such a feature. A quick research (also based on the PR you posted) makes me assume that it is very easy ..., but I still have to try.

**Comment by thetwom** [comment]
I gave it a quick try. It seems that starting with android 8.1, showing an app on the lock screen, is quite simple
```
if (Build.VERSION.SDK_INT >= 27)
            setShowWhenLocked(true)
```
in the `onCreate` method in the MainActivity. Having this enabled feels a bit weird to me, so to make this official it would have to be hidden in a setting. Still not so sure if this should really be part of the official version ...

**Comment by m0nkeyc0de** [comment]
Maybe a _Tweaks_  dialog (like `AppearanceDialog`) in the _Others_ section of the settings could be an OK place to put this setting (and also other future stuff "to be tested") : it's at the bottom of the settings so it would limit the impact of making the settings list longer.

I've no experience in developing Android applications but as I like using this app I'll give a try to implement that properly on my side and submit a PR, if it's okay for you to do so.

As for the question if it should be part of the official version or not, I tend to say that there already are too many tuner applications out there.

**Comment by thetwom** [comment]
Yes, we could kind of put the option more to the end of the preferences. I would not necessarily hide it in an extra dialog for now, one could think about this later, if too many such options appear. 

If you are interested in trying to try it by your own, let me know, then I could at least guide you where to add the different bits.
